lib/activerecord_hoarder/batch.rb

```diff
@@ -2,25 +2,44 @@ module ::ActiverecordHoarder
   class Batch
     RECORD_DATE_FIELD = "created_at"
 
-    def self.from_records(record_data)
-      record_data.present? ? new(record_data) : nil
-    end
-
-    def initialize(record_data)
+    def initialize(record_data, delete_transaction: nil)
       @record_data = record_data
       @serializer = ::ActiverecordHoarder::Serializer
+      @delete_transaction = delete_transaction
+    end
+
+    def content_string
+      @serializer.create_archive(@record_data)
```

Comment:
Could have a better name - don't believe the @serializer is actually creating an archive here in terms of how I would define creating.

Reply:
renamed to serialize